src/cargo/core/features.rs

```diff
@@ -929,6 +940,10 @@ impl CliUnstable {
         for flag in flags {
             self.add(flag, &mut warnings)?;
         }
+
+        if self.gitoxide.is_none() && option_env!("CARGO_USE_GITOXIDE_INSTEAD_OF_GIT2").is_some() {
```

Comment:
I'm assuming his is meant to be an internal-only env variable used for testing.  It should probably be renamed to `__CARGO_USE_GITOXIDE_INSTEAD_OF_GIT2`


... brainstorming thought: should we skip this in release builds?

Reply:
A valid point! The rename was performed here: 393c10070907bf53b93489ff07ac960e78926e81 .

It's a probably false intuition of mine that it should be easier for local testing if this feature remains always available, but then again, it's probably just as easy to turn on using a `.cargo/config.toml` file.